### PR TITLE
New email template

### PR DESCRIPTION
**Pull request by danielhusar**
Na zaklade https://github.com/slovensko-digital/navody.digital/issues/194 som implementoval dizaj cislo 2 :)

<table>
<tr>
  <th>Confirmation email</th>
  <th>Magic link</th>
</tr>
<tr>
  <td valign ="top">

   Screen Shot 2019-05-25 at 18 29 00

  <td valign ="top">
  
Screen Shot 2019-05-25 at 18 28 47

</table>

Spravil som aj litmus test, a vyzera to ok vo vsetkom moznom:

<details>
<summary>Screenshoty</summary>

### android 8
android8_T0bkLr6WJ7T

### gmail_com_chrome
gmail_com_chrome

### gmail_com_firefox
gmail_com_firefox

### gmail_com_internet_explorer
gmail_com_internet_explorer

### iPad5
iPad5

### iPhone8
iPhone8

### MacOSX_mail12
MacOSX_mail12

### MacOSX_outlook2019
MacOSX_outlook2019

### outlook_live_com_firefox
outlook_live_com_firefox

### outlook_office365_com_internet_explorer
outlook_office365_com_internet_explorer

### win_nativeMail10
win_nativeMail10

### win_outlook2019
win_outlook2019

### win_thunderbird
win_thunderbird

</details>

fixes https://github.com/slovensko-digital/navody.digital/issues/194

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -32,7 +32,7 @@
   # `config.assets.precompile` and `config.assets.version` have moved to config/initializers/assets.rb
 
   # Enable serving of images, stylesheets, and JavaScripts from an asset server.
-  # config.action_controller.asset_host = 'http://assets.example.com'
+  config.action_controller.asset_host = 'https://navody.digital'
```
Comment (danielhusar) [inline config/environments/production.rb]: Url musia byt absolutne v mailoch, ak by niekto pouzil ActionView helper ako napriklad `image_tag`.

File: app/views/notification_subscription_mailer/confirmation_email.html.erb [hunk 2]
```diff
@@ -1,5 +1,9 @@
-<p>Pre potvrdenie zasielania notifikácii na stránke Návody.Digital kliknite na nasledujúci odkaz.</p>
+<div class="center">
+  <p>Pre potvrdenie zasielania notifikácii na stránke Návody.Digital kliknite na nasledujúci odkaz.</p>
 
-<%= link_to 'Potvrdzujem zasielanie notifikácii', confirm_notification_subscription_group_url(@token) %>
+  <%= render "components/email_btn" do %>
```
Comment (danielhusar) [inline app/views/notification_subscription_mailer/confirmation_email.html.erb]: Bohuzial v mailoch je stale tabulka najpolahlivejsie riesenie na tlacidla.

File: Gemfile [hunk 3]
```diff
@@ -52,7 +52,7 @@ gem 'pg_search'
 gem 'kaminari'
 gem 'exception_handler'
 
-
+gem 'premailer-rails'
```
Comment (danielhusar) [inline Gemfile]: premailer inline vsetky css rules ako inline

**Comment by danielhusar** [comment]
BTW zelene tlacidla nesplnaju 4.5 :1 contrast ratio:
Screen Shot 2019-05-25 at 18 57 01

File: config/environments/production.rb (hunk 1 above)
Comment (jsuchal) [reply config/environments/production.rb]: Toto by som asi zobral z ENV, lebo mame aj staging env s url staging.navody.digital

Comment (danielhusar) [reply config/environments/production.rb]: Hotovo :)

Inac ja by so presunul `app.yml` do `secrets.yml`, je to trosku jednoduchsie ziskat config potom.

File: config/environments/production.rb [hunk 4]
```diff
@@ -32,7 +33,7 @@
   # `config.assets.precompile` and `config.assets.version` have moved to config/initializers/assets.rb
 
   # Enable serving of images, stylesheets, and JavaScripts from an asset server.
-  config.action_controller.asset_host = 'https://navody.digital'
+  config.action_controller.asset_host = app_config["host"]
```
Comment (danielhusar) [inline config/environments/production.rb]: treba tu manualne dat aj protocol?

**Pull request merged**